composer.json

```diff
@@ -36,5 +36,10 @@
         "psr-4": {
             "Riskio\\IdempotencyModule\\": "src/"
         }
+    },
+    "extra": {
+        "zf": {
+            "module": "Riskio\\IdempotencyModule\\Module"
+        }

```

Comment:
The `ConfigProvider` class would just need to return whatever is in the array of config (as per my understanding of the documentation)